test/framework/deployment.go

```diff
@@ -83,10 +84,13 @@ func (f *Framework) DeleteDeployment(ns, name string) error {
 
 // WaitUntilDeploymentReady waits until given deployment is ready.
 func (f *Framework) WaitUntilDeploymentReady(ns, name string) error {
-	err := wait.Poll(5*time.Second, f.DefaultTimeout, func() (bool, error) {
+	err := wait.Poll(time.Second, f.DefaultTimeout, func() (bool, error) {
```

Comment:
is retrying every second what you think works best, or what is the reason behind this change?

Reply:
Yes. With the new changes, I keep recreating the exporter, thus I felt that I needed to reduce the polling interval in order to optimize the setup time of each test.